libs/blocks/ost/ost.js

```diff
@@ -0,0 +1,48 @@
+import { loadScript, loadStyle } from '../../utils/utils.js';
+
+const IMS_COMMERCE_CLIENT_ID = 'aos_milo_commerce';
+const IMS_PROD_URL = 'https://auth.services.adobe.com/imslib/imslib.min.js';
+const OST_SCRIPT_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/index.js';
+const OST_STYLE_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/index.css';
```

Comment:
since we load OST styles externally, we can remove blocks/ost/ost.css

Reply:
Removed